components/button-cards/contact-card/contact-card.scss

```diff
@@ -180,3 +174,22 @@
     z-index: 1;
   }
 }
+
+@media (min-width: 769px) and (max-width: 992px) {
```

Comment:
I would recommend to move the media query to the relevant element. In this case you have to scroll multiple times in the file to get the complete overview over an element. Which i think is not ideal